libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -211,6 +214,15 @@ class AP_InertialSensor : AP_AccelCal_Client
     // get the accel filter rate in Hz
     uint16_t get_accel_filter_hz(void) const { return _accel_filter_cutoff; }
 
+    // harmonic notch current center frequency from IMU 1
+    float get_gyro_harmonic_notch_center_freq_hz(void) const { return _calculated_harmonic_notch_freq_hz; }
+
+    // harmonic notch reference center frequency
+    float get_gyro_harmonic_notch_reference_freq_hz(void) const { return _harmonic_notch_filter.center_freq_hz(); }
+
+    // harmonic notch reference scale factor
+    float get_gyro_harmonic_notch_reference(void) const { return _harmonic_notch_filter.reference(); }
```

Comment:
Are the 3 methods above going to be used soon? They are unused at the moment (as is the reference scale factor).

Reply:
Yes all used when we wire to throttle

Reply:
It's a bit of a golden rule in AP that we don't add things before they are used so let's remove these.  Otherwise you can just look forward to a PR from muramura in 2 weeks removing them :-).